third_party/opa/data-and-policies/sample_policies.rego

```diff
@@ -37,7 +37,7 @@ deny[action] {
 }
 
 deny[action] {
-	description = "deny if role is not Business Analyst when purpose is Customer Behaviour Analysis"
+	description = "Deny if role is not Business Analyst when purpose is Customer Behaviour Analysis"
```

Comment:
Check that the processing geography is not Turkey.

Reply:
Added the rule you mentioned @simanadler